library/Core.cpp

```diff
@@ -1481,11 +1481,12 @@ bool Core::Init()
     // make it obvious what's going on if someone checks the *.txt files.
     #ifndef LINUX_BUILD
         // Don't do this on Linux because it will break PRINT_MODE:TEXT
+        // this is handled as appropriate in Console-posix.cpp
         fprintf(stdout, "dfhack: redirecting stdout to stdout.log (again)\n");
-        fprintf(stderr, "dfhack: redirecting stderr to stderr.log (again)\n");
         freopen("stdout.log", "w", stdout);
-        freopen("stderr.log", "w", stderr);
     #endif
+    fprintf(stderr, "dfhack: redirecting stderr to stderr.log (again)\n");
```

Comment:
The `(again)` was only for Windows.

Also, if you're moving this logic here, you should also remove it from `Hooks-darwin`, like you removed it from `Hooks-linux`

Reply:
done